src/lib/arch/aarch64/Instruction_execute.cc

```diff
@@ -2,6 +2,11 @@
 #include <limits>
 #include <tuple>
 
+// Temporary; until execute has been verified to work correctly.
+#ifndef NDEBUG
+#include <iostream>
+#endif
+
```

Comment:
Still needed?

Reply:
I think so. I have only tested it on my end, so it would be better if the team could test as well.